config/nf_modules.config

```diff
@@ -102,20 +114,20 @@ process {
             [
                 path: "${params.outdir}/",
                 mode: params.publish_dir_mode,
-            ],
+            ]
         ]
     }
 
     withName: "CONCOCT_.*" {
         publishDir = [
             [
-                path: { "${params.outdir}/intermediate_steps/Binning/CONCOCT/bins" },
+                path: { "${params.outdir}/intermediate_steps/binning/concot/bins" },
                 mode: params.publish_dir_mode,
                 saveAs: { filename -> new File(filename).getName() },
                 pattern: "*/*.fa"
             ]
         ]
-        ext.prefix = { "CONCOCT-${meta.id}" }
```

Comment:
That should be CONCOCT as it is name of [tool](https://github.com/BinPro/CONCOCT) and we will grep name of the tool from bin filename and submit those values to ENA.